docs/settings/alert-action-settings.asciidoc

```diff
@@ -245,6 +245,8 @@ Specifies the maximum number of actions that a rule can generate each time detec
 
 `xpack.alerting.rules.run.alerts.max` {ess-icon}::
 Specifies the maximum number of alerts that a rule can generate each time detection checks run. Default: 1000.
++
+WARNING: Configuring your alerting system to handle more than 1000 alerts per run requires careful consideration and is not supported. Doing so could strain system resources, leading to performance issues, delays in alert processing, and potential disruptions during high alert activity periods.
```

Comment:
I quite like the wording if the "important" note in https://www.elastic.co/guide/en/elasticsearch/reference/8.9/misc-cluster-settings.html#cluster-shard-limit so I've poached some of that in this suggestion:

```suggestion
WARNING: The exact number of alerts your cluster can safely handle depends on your cluster configuration and workload, however setting a value higher than the default (`1000`) is not recommended or supported. Doing so could strain system resources and lead to performance issues, delays in alert processing, and potential disruptions during high alert activity periods.
```

Alternatively, we could pare it down to just this:
> 
WARNING:  Values higher than the default (`1000`) are not supported. Doing so could strain system resources and lead to performance issues, delays in alert processing, and potential disruptions during high alert activity periods.

Reply:
I like the initial proposal, I think I will go with that